login.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://kind-agnesi-dfedbe.netlify.app/component.css">
+    <link rel="stylesheet" href="home.css">
+    <link rel="stylesheet" href="login.css">
+    <script src="https://kit.fontawesome.com/95e21f3074.js" crossorigin="anonymous"></script>
+    <title>ScroleKart</title>
+</head>
+
+<body>
+    <div class="ec-login-container">
+        <header class="ec-navbar-wrapper">
+            <div class="ec-icon-div">
+                <img class="ec-main-icon" src="image/scrolekart logo.svg" alt="logo">
+            </div>
+            <nav class="ec-navbar">
+                <ul class="ec-nav-link">
+                    <li><input class="input-box-basic ec-input" placeholder="Looking for something ?" type="text">
+                        <label for="Email"></label></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="product.html">Product</a></li>
+                    <li><a href="login.html">Login/Signup</a></li>
+                    <li><a class="cart-icons" href="cart.html"><i class="fa-solid fa-cart-shopping"></i></a></li>
+                    <li><a class="cart-icons" href="wishlist.html"><i class="fa-solid fa-heart"></i></a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class="ec-login-wrapper">
+            <div>
+                <form class="ec-login-form" action="">
+                    <div class="h3 form-text-head">Login!</div>
+                    <hr>
+                    <div class="h5">Email address*</div>
+                    <div class="component-input login-input">
+                        <input class="input-box-basic" placeholder="Enter email-id" type="text">
+                        <label for="Email"></label>
```

Comment:
label is not connected to input, you have to use id of input in for attribute of label, to connect them.
Or another way is to place input field inside label itself.

Reply:
Fixed!